### PR TITLE
Adding Gem rock tracking to the Mining Plugin.

### LINKED ISSUE #9314: Gem Rock tracking in the Mining Add-on
It would be nice if this add-on tracked gem rocks as well.

Thanks.

### PR DESCRIPTION
**Pull request by Dava96**
closes #9314

### CODE REVIEW

**Comment by Alexsuperfly** [comment]
at this point might as well wildcard import object ids

File: runelite-client/src/main/java/net/runelite/client/plugins/mining/Rock.java [hunk 1]
```diff
@@ -29,27 +29,8 @@
...
-import static net.runelite.api.ObjectID.ROCKS_11372;
-import static net.runelite.api.ObjectID.ROCKS_11373;
-import static net.runelite.api.ObjectID.ROCKS_11374;
-import static net.runelite.api.ObjectID.ROCKS_11375;
-import static net.runelite.api.ObjectID.ROCKS_11376;
-import static net.runelite.api.ObjectID.ROCKS_11377;
-import static net.runelite.api.ObjectID.ROCKS_11386;
-import static net.runelite.api.ObjectID.ROCKS_11387;
-import static net.runelite.api.ObjectID.ASH_PILE;
+
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/mining/Rock.java]: remove this empty line. Also, import RL codestyle to your IDE and reorganize imports: https://github.com/runelite/runelite/wiki/Code-Conventions

**Comment by Adam-** [comment]
Where is this time value from? The wiki says ~105 seconds

**Comment by Adam-** [comment]
I checked and the 1 minute time seems correct. I also updated the wiki.

**Pull request merged**